docs/source/config_files.rst

```diff
@@ -1,9 +1,32 @@
 .. _config-files:
 
-Supported configuration files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
 
-Below is a list of supported configuration files (roughly in the order of build priority).
+``repo2docker`` looks for configuration files in the repository being built
+to determine how to build it. It is philosophically similar to
+`Heroku Build Packs <https://devcenter.heroku.com/articles/buildpacks>`_.
+
+In general, ``repo2docker`` uses the same configuration files as other software
+installation tools, rather than creating new custom configuration files.
+These files are described in :ref:`config-files`.
+
+``repo2docker`` configuration files are **composable**- you can use any number
+of them in the same repository.
+
+``repo2docker`` will look for configuration files in either:
+
+* A folder named ``binder`` in the root of the repository.
+* The root of the repository.
```

Comment:
-> "...repository or the root..."?

Reply:
I think in this case bullets make things more clear since it suggests prioritization